### ISSUE TITLE
click on column name not pop menu if column name contains '; / ? : @ & = + $ ,'

### ISSUE THREAD

**Issue by TC500** [man-group/dtale#467]
show df with column name contains '; / ? : @ & = + $ ,'
```python
import pandas as pd
import dtale
import dtale.app as dtale_app
dtale_app.JUPYTER_SERVER_PROXY = True
PROXY_ROOT='/mtjupyter/user/xxx/'
df = pd.DataFrame({'f-{"name":"test","para":{"name":"test?"}}':[1,2,3]})
dtale.show(df,notebook=True,ignore_duplicate=True,app_root=PROXY_ROOT)
```
before click:
![image](https://user-images.githubusercontent.com/13390065/115876685-36a96000-a479-11eb-8504-1e33098f5c75.png)
when i click the column name, the showed table disapear, the page all white:
![image](https://user-images.githubusercontent.com/13390065/115876741-44f77c00-a479-11eb-90a4-1fe6293fd95a.png)

i found that when i click column name, the url is 'https://mlp.sankuai.com/mtjupyter/user/xxx/proxy/40000/dtale/column-filter-data/19/f-%7B%22name%22:%22test%22,%22para%22:%7B%22name%22:%22test?%22}}', it contains ':', '-' '?' and ','
you may need to generate url by function like encodeURI()

**Comment by TC500** [comment]
if column name contains whitespace like ''f-{"name":"test", "para":{"name":"test?"}}', the url is 'https://mlp.sankuai.com/mtjupyter/user/xxx/proxy/40000/dtale/column-filter-data/22/f-%7B%22name%22:%22test%22,%20%22para%22:%7B%22name%22:%22test?%22}}'
notebook show error message '2021-04-23 21:30:44,001 - ERROR    - 127.0.0.1 - - [23/Apr/2021 21:30:44] code 400, message Bad request syntax ('GET /dtale/column-filter-data/22/f-{"name":"test", "para":{"name":"test?%22}} HTTP/1.1')'

**Comment by aschonfeld** [comment]
@TC500 thanks so much for the high level of detail on this bug.  I think know how to handle this.  Will update as I get closer to a release.

**Comment by aschonfeld** [comment]
Took a little doing but I got it working, will update when a release is made

https://user-images.githubusercontent.com/11547371/115971911-53957000-a519-11eb-9db6-64277f5482c9.mov

**Comment by aschonfeld** [comment]
Just released v1.44.0 with this fix.  When you get get a sec, please toss your ⭐ on the repo for me 🙏
